pyweek36/constants.py

```diff
@@ -67,3 +67,5 @@ class InputType(IntEnum):
 DARKMATTER_TEXTURE = arcade.load_texture(
     ASSETS_DIR / "sprites" / "map" / "dark_matter.png"
 )
+
+CAMERA_DAMPING = 0.8
```

Comment:
I think 0.8 is quite hard to notice, from my testing somewhere around 0.15-0.3 feels a lot more natural